Toolset/libraries/revcommonlibrary.livecodescript

```diff
@@ -1418,6 +1436,12 @@ private function _revEnvironmentPlatform
       case "Win32"
          put "Windows" into tPlatform
          break
+      case "iphone"
```

Comment:
we should add in html5 while we are at it.

Reply:
will do